server/services/locationsService.ts

```diff
@@ -63,10 +63,18 @@ export default class LocationsService {
     return this.locationsApiClient.locations.getResidentialSummary(token, { prisonId, parentLocationId: locationId })
   }
 
+  async getSignedOperationalCapacity(token: string, prisonId: string) {
+    return this.locationsApiClient.signedOperationalCapacity.get(token, { prisonId })
+  }
+
   async getSpecialistCellType(token: string, key: string) {
     return (await this.getConstantDataMap(token, 'getSpecialistCellTypes'))[key] || 'Unknown'
   }
 
+  async getSpecialistCellTypes(token: string) {
+    return (await this.locationsApiClient.constants.getSpecialistCellTypes(token)).specialistCellTypes
```

Comment:
```suggestion
    return await this.getConstantDataMap(token, 'getSpecialistCellTypes')
    ```

Reply:
That causes a type error in `server/controllers/setCellType/index.ts` because `constantDataMaps` returns the type `{ [key: string]: { [key: string]: string } }` but these constants are in a different format to the other one and they are objects, not strings.

Reply:
Ah, I missed that the return type of `getSpecialistCellTypes` had changed, does the call to  `getSpecialistCellType` in the location decorator need to be changed to accommodate?

Reply:
Only if it needs to know the hint text that comes from `additionalInfo`